src/components/Layout/Header/Header.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FC, useState } from "react";
+import { Link } from "react-router-dom";
+
+import styled from "@emotion/styled";
+
+import { strings } from "../../../assets/LocalizedStrings";
+import { screenWidths } from "../../../styles/mediaBreakpoints";
+
+import "@councildataproject/cdp-design/dist/images.css";
+
+const CDPLogo = styled.div({
+  float: "left",
+  marginRight: "48px",
+});
```

Comment:
For some reason this isn't rendering for me.... Tried both firefox and chrome. Any ideas?